### DISCUSSION TITLE
How to make Faster Whisper translations less polite?

### DISCUSSION THREAD

**Discussion by SteveMurgaski** [Purfview/whisper-standalone-win#328] [Q&A] [answered]
I translated the audio of a short Youtube video from French to English, using both Faster Whisper and a different program, then compared the results. The other program is the sound editor Goldwave, which recently added a transcription/translation tool that also uses Open AI's Whisper models. I was expecting similar results because they use the same models, and because the French audio in the video is very clear.
But one striking difference in the translations is that the version produced by Goldwave contains some profanity. Based on the context I think the profanity would be accurate.
If someone wants to try and replicate this I can post more details. But does Faster Whisper have a high threshold for putting rude words in its translations, and is there a way to make it less polite?

**Comment by mjamil85** [comment]
Goldwave. Got it, and I am going to check it out for this one. Thanks.

**Comment by Purfview** [comment, marked as answer]
> I was expecting similar results

That is wrong expectation, Whisper model is non deterministic, every run can give you different results, plus a single byte change in audio can trigger very different results.

> But does Faster Whisper have a high threshold for putting rude words in its translations

There is no such thing.
